src/kv/value/internal/cast/into_primitive.rs

```diff
@@ -0,0 +1,159 @@
+#[cfg(all(src_build, feature = "kv_unstable_const_primitive"))]
+pub(super) fn into_primitive<'v, T: 'static>(value: &'v T) -> Option<crate::kv::value::internal::Primitive<'v>> {
+    use std::any::TypeId;
+
+    use crate::kv::value::internal::Primitive;
+    
+    trait ToPrimitive where Self: 'static {
```

Comment:
So strangely enough this approach using const seems slightly slower than the runtime binary search. I haven’t looked into why that is yet.

Reply:
Just a few missing inlines.